lib/session.js

```diff
@@ -395,6 +399,7 @@ export class Http3WTSession {
           visitor.onClose(args)
           break
         case 'DatagramReceived':
+          // eslint-disable-next-line no-prototype-builtins
           if (visitor && args.hasOwnProperty('datagram'))
```

Comment:
This should pacify the linter:

```suggestion
          if (visitor && Object.prototype.hasOwnProperty.call(args, 'datagram'))
```